apps/server/lib/lexical/server/code_intelligence/completion/translations/callable.ex

```diff
@@ -101,7 +104,7 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Callable do
 
   defp maybe_boost(item, %_{name: name}, default_boost \\ 5) do
     if String.starts_with?(name, "__") or name in @default_functions do
-      item
+      Builder.boost(item, 0)
```

Comment:
One of the changes I am not proud of is having to do this here; I needed to add the default boost back here because overriding sort_text essentially stripped the one that came from Builder.snippet. 

I would love to hear any suggestions you might have

Reply:
the `set_sort_text` should call `boost(0)`

Reply:
Thanks, that makes sense 👍

Reply:
My bad; I remembered my issue with that approach. If I do a boost(0) there, later maybe_boost function will prepend the already boosted sort_text, and it ends up with something like "094_099_function_name:arity" because boost also updates the sort_text of the item.